app/src/main/res/values/component_colors.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="text_input_edit_text_cursor_color">@color/primary_text_color</color>
+  <color name="toolbar_layout_background_color">@color/oppia_primary</color>
+  <!--Themes.xml-->
+  <color name="oppia_themes_status_bar_color">@color/oppia_primary_dark</color>
+  <color name="oppia_action_bar_theme_background_color">@color/oppia_primary</color>
+  <!--Styles.xml-->
+  <color name="text_view_text_color">@color/primary_text_color</color>
+  <color name="text_view_heading_text_color">@color/highlighted_text_color</color>
+  <color name="text_input_layout_text_color">@color/primary_text_color</color>
+  <color name="text_input_layout_stroke_color">@color/primary_text_color</color>
+  <color name="text_input_edit_text_text_color">@color/primary_text_color</color>
+  <color name="input_interaction_edit_text_text_color">@color/primary_text_color</color>
+  <color name="text_input_layout_background_color">@color/text_input_background</color>
+  <color name="text_input_layout_error_color">@color/error_color</color>
```

Comment:
I think I had a comment for these at one point, but I can't find it. These are too generic for component colors and feel closer to color_palette where this file should have more specific ones (like home_activity_toolbar_color or answer_box_error_color).

Reply:
These colors are not used by any components or any layout file directly. Like `toolbar_layout_background_color` is only used in `toolbar.xml` which is in turn included in other layout files. Similar is with other colors as they are only used once in some styles defined in `styles.xml` which is then in turn used for other layouts. So I don't think these can be named any more specific than the styles they are used for. Other option can be to move these to `color_palette.xml` as it is if that makes more sense than keeping them in `component_colors.xml`. So, should I move them to color_palette instead?

Reply:
Ah, that's interesting. Hmm, I think this needs to stay as a component color but perhaps we can make it clearer that it's not specific to any particular screen, e.g.: ``shared_activity_toolbar_color``. That color could still point to a color_palette ``toolbar_color`` for theming. Any thoughts @rt4914?

My main thinking is that it's probably more important to ensure that only component colors are used in layouts, and are as specific as possible, and that they tie back to color_palette so that day/night mode works easily.

Reply:
Done.